data-raw/openAPI_to_csv.pl

```diff
@@ -0,0 +1,193 @@
+#!/usr/local/bin/perl

```

Comment:
Can you summarize what's changed here from the R script?

Reply:
The R script parses my copy of the OpenAPI definition from the previous iteration.  The perl script parses their updated definition directly https://search.patentsview.org/static/openapi.json.  Just changing the url in the R script throws the error below and I wasn't able to fix it.

Error in rownames_to_column(.) : is.data.frame(df) is not TRUE
Calls: extract_relevant_schema_info ... mutate -> setNames -> rownames_to_column -> stopifnot
Execution halted